deps/wazuh_testing/wazuh_testing/wazuh.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+#
+# wazuh.py
+#
+# General information about installed Wazuh instance
+
+import platform
+import os
+from wazuh_testing.tools import WAZUH_PATH, LOG_FILE_PATH
+from subprocess import check_output, PIPE
+
+
+def get_version():
+    system_platform = platform.system()
+    version = None
+
+    if system_platform in ['Windows', 'win32']:
+        with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
+            version = f.read()
+            version = version[:version.rfind('\n')]
+
+    else:  # Linux, sunos5, darwin, aix...
+        version = check_output([
+          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"
+        ], stderr=PIPE).decode('utf-8')
+
+    return version
+
+
+def get_service():
+    system_platform = platform.system()
+    service = None
+
+    if system_platform in ['Windows', 'win32']:
+        service = 'agent'
```

Comment:
```suggestion
        return 'agent'
```
`agent` or `wazuh-agent`?

Reply:
`wazuh-agent`